docs/publish.md

```diff
@@ -0,0 +1,7 @@
+# Publishing to zapier for Kontent.ai developers
+
+1. To login with zapier-cli, use prepared command `npm run zapier-login` and fill in your credentials.
+1. This should create a deploy key will create a file `.zappierrc` with it. Now you should be ready to use `npm run buildAndPush`. Don't worry, this will push private version. If it failed due to deploy key reason, continue with next step.
```

Comment:
It will push the version you specify in the `package.json` that can but doesn't have to be private. (It is private by default if the version doesn't exist yet in Zapier.) What happens if you try to push an existing already public version? Will the command fail or will you overwrite it? It would be nice to know this and mention it here.